src/test/java/integration/PostcodePoolIT.java

```diff
@@ -143,9 +146,9 @@ public void updatePostcodePool() throws Exception {
         PostcodePool updatedPostcodePool = postcodePoolService.getEntityByUuid(
                 postcodePoolUuid);
         PostcodePoolDto postcodePoolDto = postcodePoolMapper.toDto(updatedPostcodePool);
-        long timeModified = updatedPostcodePool.getLastModified().getTime();
+        LocalDateTime timeModified = updatedPostcodePool.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).